src/layouts/Footer/Footer.style.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file Footer.style.js
+ * Exports Footer component styles.
+ */
+
+export default () => ({
+  wrapper: {
```

Comment:
Styles for components are now exported from `{ComponentName}.style.js`. This will save us from having giant style definitions in component files.